wordle-web/src/views/WordleView.vue

```diff
@@ -1,4 +1,10 @@
 <template>
+  <GameScore :username="username" @restartGame="restartGame" :gameState="gameState" :game="game" />
+  <UserName
+    @sendUsername="setUsername"
+    @disableKeyboard="setKeyboard(false)"
+    @enableKeyboard="setKeyboard(true)"
```

Comment:
Why not use one return that returns a boolean?

Reply:
Fixed, thank you